virttest/shared/control/install_pkg.control

```diff
@@ -13,9 +13,9 @@ It using --args for passing package names.
 
 ..../autotest-local --args="gcc make"
 '''
-from avocado.utils import software_manager
```

Comment:
I think we can skip this file since it is no longer used. Please see https://github.com/avocado-framework/avocado-vt/pull/3205 for the details.

Reply:
Done, thanks for informing me.